pages/hydra/1_new_start_here/other-getting-started.md

```diff
@@ -4,7 +4,23 @@ keywords: Dive Into Hydra Works
 permalink: /other-getting-started.html
 folder: hydra/getting_started/other-getting-started-docs.md
 tags: [getting_started]
+toc: false
 ---
 
+- [Hyrax ](https://github.com/projecthydra-labs/hyrax#getting-started)
 
-{% include links.html %}
+- [Solr](http://lucene.apache.org/solr/quickstart.html)
+
+- [Fedora 4 ](https://wiki.duraspace.org/display/FEDORA471/Quick+Start)
+
+- [Blacklight](https://github.com/projectblacklight/blacklight/wiki/Quickstart)
+
+- [Rails ](http://guides.rubyonrails.org/getting_started.html)
+
+- [AWS](https://aws.amazon.com/getting-started/)
```

Comment:
They seemed like good candidates for things tangentially related to the stack, but not highly useful here either. Since they're really striking you as out of context and not as extra niceties, I'm persuaded it's better to get rid of them.